goal_src/engine/math/vector-h.gc

```diff
@@ -493,8 +493,11 @@
     (set! (-> ,v x) ,xv)
     (set! (-> ,v y) ,yv)
     (set! (-> ,v z) ,zv)
-    (set! (-> ,v w) ,wv))
-  )
+    (set! (-> ,v w) ,wv)))
+
+(defmacro print-vector! (v)
```

Comment:
I think they used `!` only for functions which mutate the input.
Also, in newer versions of the compiler, you can just do `(print x)` and it will actually call the print method.

Reply:
Do vectors have a print method? I forgot I even made this change, was back when i was doing some initial tests, might not be relevant if there's a simpler way to print a vector as you mention

Reply:
Yep: https://github.com/water111/jak-project/blob/master/goal_src/engine/math/vector-h.gc#L319
but it used to be annoying to call due to `print` being the name of both a method (which does the right thing) and a function (which doesn't).  Now the method should win over the function.